addons/kommander/1.1.x/kommander.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: kommander
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-18"
-    appversion.kubeaddons.mesosphere.io/kommander: "1.1.0-rc.1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-19"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.1.0-rc.2"
```

Comment:
```suggestion
```
… dont just increase all numbers that show up 🙈 😅 its still rc.1

Reply:
I don't understand that. We're obviously changing functionality. Why should the version number stay the same?

Reply:
I would actually suppose we **always** have to increase both version numbers whenever we bump the chart.

Reply:
kommander does not release on every merge to master, that version is going to be the next release we do.